Doc/library/email.compat32-message.rst

```diff
@@ -660,10 +660,10 @@ Here are the methods of the :class:`Message` class:
 
       .. testsetup::
 
-         >>> from email import message_from_binary_file
-         >>> with open('Lib/test/test_email/data/msg_16.txt', 'rb') as f:
-         ...     msg = message_from_binary_file(f)
-         >>> from email.iterators import _structure
+         from email import message_from_binary_file
+         with open('../Lib/test/test_email/data/msg_16.txt', 'rb') as f:
```

Comment:
This is obviously fragile.  Someone added the testsetup, and presumably it worked for them when they ran it and this change would break it for them.  We should compute the path relative to ``email.__file__``.  That won't work in all cases, but it will work in more cases.